addons/mail/static/src/components/composer/composer.js

```diff
@@ -232,17 +232,19 @@ class Composer extends Component {
 
         const context = {
             // default_parent_id: this.id,
-            default_body: mailUtils.escapeAndCompactTextContent(this.composer.textInputContent),
             default_attachment_ids: attachmentIds,
+            default_body: mailUtils.escapeAndCompactTextContent(this.composer.textInputContent),
+            /* FIXME would need to use suggested_partners here task-2280157 */
             // default_partner_ids: partnerIds,
             default_is_log: this.composer.isLog,
+            default_use_template: true,
```

Comment:
Why ? This should be set with a template_id set, or has no use.

Reply:
Bad c/c I think, will remove